src/endpoints/http/collections/export/lib/archive.js

```diff
@@ -0,0 +1,16 @@
+import yazl from 'yazl';
+
+/**
+ * Gets the size of each collection requested to export.
+ *
+ * @param {Stream} collections - An array of streams containing each collection's documents.
+ * @returns {Stream} - Stream containing zip entries for each collection.
+ */
+export default function archive(collections=[]) {
```

Comment:
A question here around whether we always want to send a zipped file.
If there is only one collection requested does it make more sense to just export the collection in a `json`, `bson`, `csv` file?

@wei-lee @pb82